pysb/tests/test_util.py

```diff
@@ -0,0 +1,41 @@
+from pysb.core import *
+from pysb.util import *
+from pysb.examples import earm_1_0
+
+
+def test_alias_model_components():
+    """
+    Tests that alias_model_components() exports to the global namespace
+    """
+    m = Model(_export=False)
+    m.add_component(Monomer('temp_monomer_A', _export=False))
+    try:
+        temp_monomer_A
+        raise NameError('Monomer A should not be defined in global namespace')
```

Comment:
This won't fail if temp_monomer_A *is* defined! You should use nose's `assert_raises` to ensure that expected exceptions are raised. It would look something like this:

```
from nose.tools import assert_raises
assert_raises(NameError, lambda: temp_monomer_A)
```

The trick is that the second arg to assert_raises is a callable. I used a lambda to create a callable that just evaluates the global in question, which should raise NameError.

Also, why not just call the monomer 'A'?